app/controllers/concerns/foreman/controller/registration.rb

```diff
@@ -19,18 +19,20 @@ def global_registration_vars
     host_group = Hostgroup.authorized(:view_hostgroups).find(params['hostgroup_id']) if params["hostgroup_id"].present?
     operatingsystem = Operatingsystem.authorized(:view_operatingsystems).find(params['operatingsystem_id']) if params["operatingsystem_id"].present?
 
+    context = {
+      user: User.current,
+      auth_token: User.current.jwt_token!(expiration: 4.hours.to_i),
+      organization: (organization || User.current.default_organization || User.current.my_organizations.first),
+      location: (location || User.current.default_location || User.current.my_locations.first),
+      hostgroup: host_group,
+      operatingsystem: operatingsystem,
+      url_host: registration_url.host,
+      registration_url: registration_url,
+    }
```

Comment:
Perhaps also an empty line here
```suggestion
    }

```